core/client/fs/src/main/java/alluxio/util/FileSystemOptionsUtils.java

```diff
@@ -342,10 +344,12 @@ public static SetAttributePOptions setAttributeClientDefaults(AlluxioConfigurati
     // Specifically set and override *only* the metadata sync interval
     // Setting other attributes by default will make the server think the user is intentionally
     // setting the values. Most fields withinSetAttributePOptions are set by inclusion
+    FileSystemMasterCommonPOptions.Builder builder = FileSystemMasterCommonPOptions.newBuilder();
+    if (conf.isSetByUser(PropertyKey.USER_FILE_METADATA_SYNC_INTERVAL)) {
+      builder.setSyncIntervalMs(conf.getMs(PropertyKey.USER_FILE_METADATA_SYNC_INTERVAL));
+    }
```

Comment:
don't create this obj if not set by user
```suggestion
    FileSystemMasterCommonPOptions.Builder builder;
    if (conf.isSetByUser(PropertyKey.USER_FILE_METADATA_SYNC_INTERVAL)) {
      builder = FileSystemMasterCommonPOptions.newBuilder();
      builder.setSyncIntervalMs(conf.getMs(PropertyKey.USER_FILE_METADATA_SYNC_INTERVAL));
    }
```

Reply:
@jiacheliu3 It build failed after accept this change, because `builder` not initialized, now I have to revert it.